reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb

```diff
@@ -178,33 +178,6 @@
     "    !/bin/bash ./common/setup.sh"
```

Comment:
<p>why we remove the 'Use docker image section'? We can say we are using ray version 0.8.5.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
We don't mention this in the notebooks where we use the ray-version param in the `RLEstimator` function. Just following that convention here.